### PR TITLE
[KYUUBI #4975] Fixed run test link in PR template

### PR DESCRIPTION
**Pull request by mans2singh**
### _Why are the changes needed?_
The run test link in the template is broken.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by pan3793** [comment]
@mans2singh thanks, would you mind reverting unrelated changes?

**Comment by mans2singh** [comment]
@pan3793 - 

I have updated the PR based on your review.  Please let me know if there is anything else required.

Thanks

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -20,4 +20,5 @@ Please clarify why the changes are needed. For instance,
 
 - [ ] Add screenshots for manual tests if appropriate
 
-- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request
+- [ ] [Run test](- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request
```
Comment (pan3793) [inline .github/PULL_REQUEST_TEMPLATE:23]:
```suggestion
- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request
```

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -20,4 +20,5 @@ Please clarify why the changes are needed. For instance,
 
 - [ ] Add screenshots for manual tests if appropriate
 
-- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request
+- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request
+) locally before make a pull request
```
Comment (pan3793) [inline .github/PULL_REQUEST_TEMPLATE:24]:
```suggestion
```

**Comment by pan3793** [comment]
Thanks, merged to master

**Pull request closed without merging**